docs/reference/changelog.rst

```diff
@@ -77,6 +77,40 @@ Changelog
 
   For a complete list of commits, see the `X.Y.Z`_ release on GitHub.
 
+3.2.0 (2024-08-27)
+------------------
+
+We have some fixes to the 3.1 series, as well as the features below.
+The most notable under-the-hood work is that Charmcraft now uses pydantic 2.
+
+Core
+====
+
+You can now set ``charm-user`` in ``charmcraft.yaml`` to set what user Juju 3.6.0+ will
+use for running a kubernetes charm.
+
+Plugins
+#######
+
+reactive
+""""""""
+
+Fix: ``actions.yaml`` is no longer overwritten.
+
+Extensions
+##########
+
+go-framework
+""""""""""""
+
+New ``go-framework`` extension for easily charming go applications
+
+Documentation
+=============
+
+Documentation now contains a changelog, and you're reading it right now.
```

Comment:
```suggestion
The changelog is now included in the Charmcraft documentation. For completeness, we've back-filled the log with all the important changes from previous releases documented on GitHub.
```

Cheeky, but I think we should be careful with the continuous "you're". It can sound accusatory. It's the same reason the style guide advises us not to write "allow" or "let" because it speaks as though the software has power over the user.